src/OrderFormAcceptanceTests.TestData/OrderItemList.cs

```diff
@@ -3,12 +3,13 @@
     using System;
     using System.Collections.Generic;
     using System.Linq;
+    using OrderFormAcceptanceTests.Domain;
 
-    public sealed class OrderItemList
+    public sealed class List
```

Comment:
Do you definitely want to rename this class to `List`? It still seems to be specific to order items and `List` is a bit vague. If you definitely want to rename it to `List` then recommend changing the file name to match too.